projects/plugins/crm/webpack.config.js

```diff
@@ -186,4 +187,58 @@ module.exports = [
 			} ),
 		],
 	},
+	{
+		entry: {
+			'onboarding-wizard': './src/js/onboarding-wizard-index.jsx',
+		},
+		mode: jetpackWebpackConfig.mode,
+		devtool: jetpackWebpackConfig.isDevelopment ? 'source-map' : false,
+		output: {
+			...jetpackWebpackConfig.output,
+			path: path.resolve( './build' ),
+		},
+		optimization: {
+			...jetpackWebpackConfig.optimization,
+		},
+		resolve: {
+			...jetpackWebpackConfig.resolve,
+		},
+		node: false,
+		plugins: [
+			...jetpackWebpackConfig.StandardPlugins( {
+				DependencyExtractionPlugin: { injectPolyfill: true },
+			} ),
+		],
+		module: {
+			strictExportPresence: true,
+			rules: [
+				// Transpile plugin JavaScript.
+				jetpackWebpackConfig.TranspileRule( {
+					exclude: /node_modules\//,
+				} ),
+
+				// Transpile @automattic/jetpack-* in node_modules too.
+				jetpackWebpackConfig.TranspileRule( {
+					includeNodeModules: [ '@automattic/jetpack-' ],
+				} ),
+
+				// Handle CSS.
+				jetpackWebpackConfig.CssRule( {
+					extensions: [ 'css', 'sass', 'scss' ],
+					extraLoaders: [ 'sass-loader' ],
+				} ),
+
+				// Handle images.
+				jetpackWebpackConfig.FileRule(),
+			],
+		},
+		externals: {
+			...jetpackWebpackConfig.externals,
+			jetpackConfig: JSON.stringify( {
+				consumer_slug: 'zero-bs-crm',
+			} ),
+			'@wordpress/i18n': 'global wpI18n',
+			'@wordpress/jp-i18n-loader': 'global jpI18nLoader',
+		},
```

Comment:
I assume we have to use the same workaround here as we do further up the config?